server/package.json

```diff
@@ -21,7 +21,8 @@
     "compression": "^1.7.4",
     "express": "^4.16.4",
     "path": "^0.12.7",
-    "serve-favicon": "^2.5.0"
+    "serve-favicon": "^2.5.0",
```

Comment:
Why are you adding serve favicon in this PR?

Reply:
when resolving the conflict, it have been removed and I added it again.